docs/source/science/ESA_CCI/ESA_CCI_V4.ipynb

```diff
@@ -17,52 +17,39 @@
     "tags": []
```

Comment:
<p>What's the value added in terms of a tutorial to print the values ? It feels quite verbose. Is it so that we can communicate the fact that we're using <code>dask</code> in the background, and the data is only processed when the operation is performed, across <code>dask</code> workers ? If that's the intent, we could simply mention that we're using <code>dask</code> in the headers?</p><p> </p><p>This comment applies to all the cells where we print the values.</p><p> </p><p>Happy to be wrong, if it's something we've typically been doing in other dataset tutorials :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>The idea behind printing values is to show the users how filtering works. Otherwise the changes occurred by <code>where</code> function is not visible to the users. </p>